test/cpp_api_parity/utils.py

```diff
@@ -0,0 +1,247 @@
+from collections import namedtuple
+import unittest
+import os
+
+import torch
+import torch.utils.cpp_extension
+import torch.testing._internal.common_nn as common_nn
+from torch.testing._internal.common_cuda import TEST_CUDA
+
```

Comment:
suggestion, It would be good if we can put a short description for the following 2 namedtuples, what is the usage of each variable inside the tuples. We do have samples to demonstrate the usage of these variables, but it did not cover all of these, for eg 'cpp_function_call', and it would same a lot of time for people who want to add new test cases.

Reply:
Done :D The main note on this is `NOTE [How to check NN module / functional API parity between Python and C++ frontends]` in common_nn.py